assets/src/components/options-context-provider/index.js

```diff
@@ -55,6 +58,52 @@ export function OptionsContextProvider( { children, optionsRestPath, populateDef
 		hasUnmounted.current = true;
 	}, [] );
 
+	/**
+	 * Fetches plugin suppression data.
+	 */
+	const refetchPluginSuppression = useCallback( () => {
+		if ( error || fetchingPluginSuppression ) {
+			return;
+		}
+
+		/**
+		 * Fetches suppression data from the REST endpoint.
+		 */
+		( async () => {
+			setFetchingPluginSuppression( true );
+
+			try {
+				const fetchedPluginSuppression = await apiFetch( {
+					path: addQueryArgs( optionsRestPath, {
+						_fields: [ 'suppressed_plugins', 'suppressible_plugins' ],
+					} ),
+				} );
+
+				if ( true === hasUnmounted.current ) {
+					return;
+				}
+
+				setOriginalOptions( {
+					...originalOptions,
+					...fetchedPluginSuppression,
+				} );
```

Comment:
This is great. I was thinking that this could have the effect of overriding a user's unsaved changes to plugin suppression, but this won't be the case the original options are updated, not the modified otpions.